src/databricks/labs/ucx/azure/resources.py

```diff
@@ -117,44 +121,134 @@ def inner() -> dict[str, str]:
 
         return _credentials
 
+    def get(self, path: str, host: str, api_version: str | None = None):
```

Comment:
This is incorrect and inconsistent refactoring.

One API client has to call one single host, that you supply in constructor.